insta/src/env.rs

```diff
@@ -430,18 +430,35 @@ pub fn snapshot_update_behavior(tool_config: &ToolConfig, unseen: bool) -> Snaps
     }
 }
 
+pub enum Workspace {
+    DetectWithCargo(&'static str),
+    UseAsIs(&'static str),
```

Comment:
Yes, this seems good!

Reply:
(Maybe we could change the names to something more literal / nouns, but it's fine atm)